web/profiles/joinup/config/install/block.block.rdfentitiesinproposedstate.yml

```diff
@@ -0,0 +1,32 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - joinup_core
+    - user
+  theme:
+    - joinup_theme
+id: rdfentitiesinproposedstate
```

Comment:
It's minor, but I have noticed that block ids can be improved manually. I would try to use underscores here: `rdf_entities_in_proposed_state`.